gramps/gen/proxy/proxybase.py

```diff
@@ -328,63 +328,68 @@ def iter_family_handles(self):
         Return an iterator over database handles, one handle for each Family in
         the database.
         """
-        return filter(self.include_family, self.db.iter_family_handles())
+        return (_ for _ in filter(self.include_family, self.db.iter_family_handles()))
```

Comment:
Filter returns a generator. It doesn't make sense to have a generator expression iterating over a filter. Did mypy complain about this?

Reply:
Yes, mypy complains:

`gramps/gen/proxy/proxybase.py:371: error: Incompatible return value type (got "filter[Never]", expected "Generator[FamilyHandle, None, None]")  [return-value]`

Reply:
Oh! I was wrong, it returns an iterator but not a generator. What about

```suggestion
        yield from filter(self.include_family, self.db.iter_family_handles())
```

Reply:
`yield from filter(self.include_family, self.db.iter_family_handles())` works.
I'll update the PR accordingly. Thanks